content/en/agent/proxy.md

```diff
@@ -31,7 +31,11 @@ Traditional web proxies are supported natively by the Agent. If you need to conn
 {{% tab "Agent v6" %}}
 
 Set different proxy servers for `https` and `http` requests in your Agent `datadog.yaml` configuration file.
-The Agent uses `https` to send data to Datadog, but integrations might use `http` to gather metrics. No matter the proxied requests, you can activate SSL on your proxy server. Below are some configuration examples for your `datadog.yaml` file:
+The Agent uses `https` to send data to Datadog, but integrations might use `http` to gather metrics. No matter the proxied requests, you can activate SSL on your proxy server. Below are some configuration examples for your `datadog.yaml` file.
+
+<div class="alert alert-warning">
+The <code>&ltHOST&gt;:&ltPORT&gt;</code> used to proxy metrics can NOT be used to proxy logs.
```

Comment:
Should we explain why? The reason being that logs are forwarded through TCP.

Reply:
@NBParis - This is already explained in the log proxy section. This is an extra warning message in the metrics section. Should I still add an explanation here?

Reply:
Well with this logic we could say that the warning is not necessary as there is a log proxy section :)

I believe that just stating why it's not possible or adding a link to the log proxy section would be helpful. As it would not be just a `not for logs, pass your way` but more a `for logs, please go there`.

What do you think?